spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
             profile::core::common
             profile::core::debugutils
             profile::core::docker
             profile::core::docker::prune
             profile::core::ni_packages
+            profile::core::x2go
+            profile::ts::hexrot
+            profile::ts::nexusctio
```

Comment:
Please write a shared example for `            profile::ts::nexusctio`

Reply:
support/spec created for nexusctio 👍🏻